src/main/ipcs/app.ts

```diff
@@ -203,6 +225,20 @@ export function buildMenus(
 
         break;
 
+      case "checkbox":
+        t = {
+          label: menu.label,
+          type: "checkbox",
+          accelerator: menu.shortcut,
+          enabled: !menu.disabled,
+          checked: menu.checked,
+          registerAccelerator,
+        };
+
+        t.click = buildClickHandler(menu.event, menu.eventInput, channel);
```

Comment:
Add a comment here explaining how a checkbox always triggers it click handler, but a radio menu *only* triggers it when not selected.